core/trino-spi/src/main/java/io/trino/spi/connector/ConnectorFactory.java

```diff
@@ -16,11 +16,22 @@
 import com.google.errorprone.annotations.CheckReturnValue;
 
 import java.util.Map;
+import java.util.Set;
 
 public interface ConnectorFactory
 {
     String getName();
 
     @CheckReturnValue
     Connector create(String catalogName, Map<String, String> config, ConnectorContext context);
+
+    /**
+     * Extracts property names from the provided set that may include security-sensitive
```

Comment:
nit: `Extracts` -> `Returns`

feels clearer to me.

`getRedactablePropertyNames` -> `getSecuritySensitivePropertyNames`? (to align with Airlift's `ConfigurationMetadata#isSecuritySensitive` and `@ConfigSecuritySensitive`)? WDYT?

Reply:
I had the exact same thought about renaming to security sensitive